docs/kratos/manage-identities/25_import-user-accounts-identities.mdx

```diff
@@ -645,3 +645,48 @@ The service will respond with the two identity IDs created:
   ]
 }
 ```
+
+### Errors during bulk import
+
+Failure to import an identity will not fail the whole bulk import. In the response, `"action": "error"` indicates thath the
+identity with the corresponding `patch_id` could not be imported. The `error` object contains more details about why the import
+failed.
+
+This is an example response where two identities were created and two imports failed:
+
+```json
+{
+  "identities": [
+    {
+      "action": "create",
+      "identity": "0d0ed560-43ce-42a9-bd40-aafe921c3af1",
+      "patch_id": "6086b0a8-d851-5431-91b4-b6e5e39dc88b"
+    },
+    {
+      "action": "error",
+      "patch_id": "d554dc00-49ce-5381-9bdc-79637dec85a2",
+      "error": {
+        "code": 400,
```

Comment:
How do I know which identity this is?

Reply:
You know it either from the position in the array, or from the `patch_id`.

Reply:
Ok, I found the explanation above:

```
Optionally, you can specify a patch_id property which will be returned in the response. This can be used to correlate the response with the patch.
```

Maybe make an example here how to set the patch id?

And for the position, that should be explained here as well (because just from reading it it was not clear to me).